CITATION.cff

```diff
@@ -0,0 +1,35 @@
+# https://citation-file-format.github.io/
+cff-version: 1.2.0
+message: "If you use this software, please cite it using these metadata."
+title: "Alpha: A lazy-grounding Answer-Set Programming system"
+# Authors are sorted:
+# First people, sorted by `family-names`, `given-names`.
+# Second entities, sorted by `name`.
+authors:
+- family-names: "Langowski"
+  given-names: "Michael"
+  affiliation: "TU Wien"
+  #orcid: TODO
+- family-names: "Leutgeb"
+  given-names: "Lorenz"
+  affiliation: "TU Wien"
+  orcid: "https://orcid.org/0000-0003-0391-3430"
+- family-names: "Taupe"
+  given-names: "Richard"
+  affiliation: "Siemens AG"
+  orcid: "https://orcid.org/0000-0001-7639-1616"
+- family-names: "Weinzierl"
+  given-names: "Antonius"
+  affiliation: "TU Wien"
+  orcid: "https://orcid.org/0000-0003-2040-6123"
+- name: "Siemens AG"
```

Comment:
Not sure if Siemens AG as an entity should be listed as an author, or whether it's enough to list it as the affiliation of an author. But I guess it does not hurt.

Reply:
IANAL, but my understanding is that Siemens AG is an author in the sense of copyright laws, since they did pay people to work on Alpha (and at the same time they decided that the result of that work is licensed with the BSD 2-Clause license). So the question whether it belongs here depends on what the authors list should contain: the joint copyright holders, or only persons that wrote the code.

Reply:
There's no strict definition. I'd guess the latter.